package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "1.38.4",
+  "version": "1.38.5",
```

Comment:
A new feature should increase the minor version, not the patch version. See https://semver.org/